pom.xml

```diff
@@ -679,8 +679,8 @@
         <parquet.hadoop.version>1.10.1</parquet.hadoop.version>
         <spark.version.classifier>spark${buildver}</spark.version.classifier>
         <cuda.version>cuda11</cuda.version>
-        <spark-rapids-jni.version>23.08.0-SNAPSHOT</spark-rapids-jni.version>
-        <spark-rapids-private.version>23.08.0-SNAPSHOT</spark-rapids-private.version>
+        <spark-rapids-jni.version>${project.version}</spark-rapids-jni.version>
+        <spark-rapids-private.version>${project.version}</spark-rapids-private.version>
```

Comment:
This change is not desired.  We change these versions separate from the project version (e.g.: these dependencies stop being a snapshot before this project does).